core/src/main/java/org/apache/iceberg/RemoveSnapshots.java

```diff
@@ -69,7 +69,7 @@ class RemoveSnapshots implements ExpireSnapshots {
   private final Set<Long> idsToRemove = Sets.newHashSet();
   private final long now;
   private final long defaultMaxRefAgeMs;
-  private boolean cleanExpiredFiles = true;
+  private final CleanupLevel defaultCleanupLevel = CleanupLevel.ALL;
```

Comment:
I don't know if we need this variable, just default `cleanupLevel` to `ALL`.

Reply:
To me, it provides some semantic understanding for default clean up level, as it currently default to ALL.